Yosemite/Yosemite/Stores/InAppPurchaseStore.swift

```diff
@@ -79,15 +79,31 @@ private extension InAppPurchaseStore {
     }
 
     func handleCompletedTransaction(_ result: VerificationResult<StoreKit.Transaction>) async throws {
-        switch result {
-        case .verified(let transaction):
-            logInfo("Verified transaction \(transaction.id) (Original ID: \(transaction.originalID)) for product \(transaction.productID)")
-            try await submitTransaction(transaction)
-            await transaction.finish()
-        case .unverified:
+        guard case .verified(let transaction) = result else {
+            // Ignore unverified transactions.
```

Comment:
Will be handle the unverified case in another PR?

Reply:
Honestly, I'm not sure. Apple's [sample code](https://developer.apple.com/documentation/storekit/transaction/3851206-updates) would just ignore those, but I don't fully understand in which scenario can we get an unverified transaction and if it's safe to ignore them.

My understanding is that an unverified transaction would mean that the transaction has been manipulated and can't be trusted, and we'd ignore it unless we want to log that for fraud detection purposes, but I haven't seen enough documentation to back up that understanding.